docs/howto/multiple-storage-backends.md

```diff
@@ -200,9 +206,126 @@ configurations. In multi-storage setups, this is the recommended method to list
 | Missing `backward_compatible`                                       | Upgrade from single to multi-storage without setting the flag | Add `backward_compatible: true` for the existing storage |
 | Unsupported configurations in OSS or unlicensed Enterprise accounts | Using multi-storage features in an unsupported setup | Contact us to start using the feature                    |
 
+### Migrating from Multi-Storage Backend to Single Storage Backend
+
+Once you upgrade to a multi-storage setup, you cannot simply revert back by changing the configuration from `blockstores` to `blockstore`. The internal repository metadata format changes to support multiple storage backends, and is not backward compatible with the single storage format. If you need to consolidate your data and revert from a multi-storage setup to a single storage backend, you'll need to perform a full migration by following these steps:
+
+#### Overview
+
+The migration process involves:
+
+1. Dumping repository references from the multi-storage setup
+2. Deleting repositories in the multi-storage environment
+3. Configuring lakeFS with a single storage backend
+4. Copying repository data to the new single storage location (if needed)
+5. Restoring repositories to the new lakeFS instance
+
+#### Step-by-Step Guide
+
+1. **Dump Repository References**
+
+   Use the `lakefs-refs.py` script, instruction on how to aquire found in [Backup and Resource](backup-and-restore.md#python-script-for-refs-management) to dump repository metadata:
```

Comment:
```suggestion
   Use the `lakefs-refs.py` script, instruction on how to aquire found in [Backup and Restore](backup-and-restore.md#python-script-for-refs-management) to dump repository metadata:
```

Reply:
you can move this line to be the first line under the Step-by-Step Guide heading
> instruction on how to aquire found in [Backup and Resource](backup-and-restore.md#python-script-for-refs-management)